cmd/lakectl/cmd/root.go

```diff
@@ -216,25 +216,49 @@ func withSyncFlags(cmd *cobra.Command) {
 	withNoProgress(cmd)
 }
 
+func getStorageConfig(ctx context.Context, client *apigen.ClientWithResponses, repositoryID string) (*apigen.StorageConfig, error) {
+	confResp, err := client.GetConfigWithResponse(ctx)
+	DieOnErrorOrUnexpectedStatusCode(confResp, err, http.StatusOK)
```

Comment:
This function is confusing, it returns an error, but never really returns an error, it uses DIE instead.
Please return error instead of Dying on errors

Reply:
Since the pattern in lakectl is using Die instead of errors,
Changed this one to the `get*OrDie()` pattern, used in other places as well.